### PR TITLE
Add Brightspot detection

### PR DESCRIPTION
**Pull request by rhino88**
https://www.aboutamazon.com/
https://corporate.walmart.com/
https://video.nationalgeographic.com/
https://www.sothebys.com/en/
https://www.specialolympics.org/
https://www.mybluprint.com/
https://www.sourcemedia.com/
https://www.jnj.com/
https://www.newellbrands.com/
http://www.hallmarkchannel.com/

### CODE REVIEW

**Comment by rhino88** [comment]
Hmm it seems as if the Scrutinizer task completed successfully but the status here in github does not reflect this.

File: src/apps.json [hunk 1]
```diff
@@ -1331,6 +1331,15 @@
       },
       "website": "https://www.braintreepayments.com"
     },
+    "Brightspot": {
+      "cats": [
+        1 
+      ],
+      "headers": { "X-Powered-By": "Brightspot" },
```
Comment (jvoisin) [inline src/apps.json]: Please use `"headers": { "X-Powered-By": "^Brightspot$" },` if possible

**Comment by rhino88** [comment]
Fixed. Thanks.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**